### PR TITLE
Adding enabled content experiences and compliance settings

### PR DESCRIPTION
**Pull request by emzho**
Roadmap Item ID: 14127
API.md and CSDL are both listed in the description for that roadmap item.

Documenting the enabledContentExperiences and complianceSettings properties.

### CODE REVIEW

File: api-reference/beta/resources/externalconnectors-compliancesettings.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Collects all configurable settings related to compliance over connector content.
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|eDiscoveryResultTemplates|[microsoft.graph.externalConnectors.displayTemplate](../resources/externalconnectors-displaytemplate.md) collection|Enables the developer to define the appearance of the content and configure conditions that dictate when the template should be displayed.|
+
```
Comment (emzho) [inline api-reference/beta/resources/externalconnectors-compliancesettings.md:22]: TO DO: @KumarSeetharaman74 to add a note for eDiscovery special Adaptive Card Library requirements for partners.

File: api-reference/beta/resources/externalconnectors-externalconnection.md [hunk 2]
```diff
@@ -31,16 +31,18 @@ A logical container to add content from an external source into Microsoft Graph.
...
+| Property           | Type                              | Description |
+|:--------------------------|:----------------------------------|:------------|
+| complianceSettings        |[microsoft.graph.externalConnectors.complianceSettings](../resources/externalconnectors-compliancesettings.md)| The settings configuring compliance-related content in this connection, such as the display templates for compliance results.|
+| configuration             | [microsoft.graph.externalConnectors.configuration](externalconnectors-configuration.md) | Specifies additional application IDs that are allowed to manage the connection and to index content in the connection. Optional. |
+| connectorId               | String            | The Teams App ID. Optional.|
+| description               | String           | Description of the connection displayed in the Microsoft 365 admin center. Optional. |
+| enabledContentExperiences | microsoft.graph.externalConnectors.contentExperienceType collection | The list of content experiences the connection will participate in. Possible values are `search` and `compliance`. Required. |
```
Comment (emzho) [inline api-reference/beta/resources/externalconnectors-externalconnection.md:40]: TO DO: make sure "Required" is the end decision for this property.

**Review by josmoran-MSFT — APPROVED** [review]

File: changelog/Microsoft.Search.json [hunk 3]
```diff
@@ -1197,6 +1197,32 @@
       "CreatedDateTime": "2022-04-28T00:00:00",
```
Comment (Danielabom) [inline changelog/Microsoft.Search.json:1197]: Hi @emzho, I believe that the two properties added need a changelog entry. Thanks.

**Review by Danielabom — CHANGES_REQUESTED** [review]
Hi @emzho I added a comment to the changelog file. Please review it.

File: changelog/Microsoft.Search.json (hunk 3 above)
Comment (emzho) [reply changelog/Microsoft.Search.json:1197]: Done. Thank you!

**Review by Danielabom — APPROVED** [review]
@emzho, thanks for addressing my comments. Changes look good to me.

**Review by Danielabom — APPROVED** [review]
Hi Emily, thank you for pointing me to the new change. 
Everything looks okay. Thanks.

File: api-reference/beta/resources/externalconnectors-compliancesettings.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Namespace: microsoft.graph.externalConnectors
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Collects all configurable settings related to compliance over connector content.
+
+## Properties
+
+> [!IMPORTANT]
+> The eDiscovery result template is only supported for Adaptive Card version 1.2 or below. Please refer to the Compliance settings section of [Create, update, and delete connections](/graph/connecting-external-content-manage-connections#compliance-settings) for more details.
```
Comment (emzho) [inline api-reference/beta/resources/externalconnectors-compliancesettings.md:21]: @KumarSeetharaman74 do we want to also call out the extra requirements here (not supporting markdown, not supporting data binding) or just edit it in the conceptual docs, which are linked already?

Comment (Lauragra) [inline api-reference/beta/resources/externalconnectors-compliancesettings.md]: Please verify this link; the article does have a "Compliance settings" section. Can you link directly to the section you intend?

Comment (emzho) [reply api-reference/beta/resources/externalconnectors-compliancesettings.md]: Hi, I assume you meant that it does NOT have a compliance settings section. The PR #17288 which was queued for merge also, added this section. Sorry! I guess the other one has to come first.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**